### PR TITLE
[Code health] Move LatLngConverter from model to util package

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Moves the utility class from model package to util package

@gino-m @anandwana001  PTAL?

### CODE REVIEW

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks

**Review by gino-m — CHANGES_REQUESTED** [review]
A few nits, overall LGTM!

File: ground/src/main/java/com/google/android/ground/util/CoordinatesExt.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 Google LLC
```
[the remark this answers is not in the document]
Comment (shobhitagarwal1612) [reply ground/src/main/java/com/google/android/ground/util/CoordinatesExt.kt]: Done

File: ground/src/main/java/com/google/android/ground/util/CoordinatesExt.kt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import android.location.Location
+import com.google.android.ground.model.geometry.Coordinates
+import kotlin.math.abs
+
+/** Converts the given coordinates in decimal format to D°M′S″ format. */
+fun Coordinates.toDmsFormat(): String = "${convertLatToDMS(lat)} ${convertLongToDMS(lng)}"
+
+private fun convertLatToDMS(lat: Double): String {
+  val orientation = if (lat > 0) "N" else "S"
+  return "${decimalToDMS(lat)} $orientation"
```
[the remark this answers is not in the document]
Comment (shobhitagarwal1612) [reply ground/src/main/java/com/google/android/ground/util/CoordinatesExt.kt]: Done

**Comment by gino-m** [comment]
Failed checks appear to be a flake, rerunning.

**Review by gino-m — APPROVED** [review]
👍

**Comment by shobhitagarwal1612** [comment]
I ran `checkCode` and unit tests locally. Everything is passing. Ignoring the flaky runs.

**Pull request merged**